sdk/storage/azure-storage-blob-stress/templates/stress-test-job.yaml

```diff
@@ -46,7 +46,8 @@ spec:
             -Dreactor.schedulers.defaultBoundedElasticSize={{ .Stress.parallelRuns }} \
             -jar /app/azure-storage-blob-stress-1.0.0-beta.1-jar-with-dependencies.jar \
             {{ .Stress.testScenario }} \
-            --parallel {{ .Stress.parallelRuns }} \
+            --parallel {{ default 1 .Stress.parallelRuns }} \
```

Comment:
Just a reminder of what we talked about over teams... We are going to set the default of `--parallel` to `5` and then remove the `parallelRuns` field altogether from each scenario to reduce the verbosity of the test case definitions. If for a test case we want to use something that is not `5` for `--parallel` we can then declare it in the `parallelRuns` field of the targeted test case.